tools/CAT/src/pages/editPage/editPageWithTitle/index.tsx

```diff
@@ -0,0 +1,32 @@
+import EditorWrapper from '@/components/EditorWrapper';
+import EditTitlePage from '@/components/EditTitlePage';
+
+interface EditPageDefaultProps {
+	content: string;
+	title: string;
+	textContent: string;
+	img: string;
+}
+
+export default function EditPageWithTitle({
+	title,
+	content,
+	img,
+	textContent,
+}: EditPageDefaultProps) {
+	return (
+		<div className='w-full'>
+			<EditTitlePage
+				nameLabel='template.label'
+				nameUrl='template.image.url'
```

Comment:
Can you explain these two lines?

Reply:
what exactly do I need to explain? nameLabel is the title for the page, nameUrl is a link to a photo that will be the background for the title